### PR TITLE
new methods: seed(...) and rate(...)

### PR DESCRIPTION
**Pull request by roccomuso**
@feross let me know

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,14 @@ Disconnect two nodes, identified by `id1` and `id2`, from each other.
 
 Remove a node, identified by `id`, from the graph.
 
+### graph.seed(id, seeding)
```
Comment (feross) [inline README.md:78]: boolean values are better specified starting with an `is` or `has`, for example here: `isSeeding`

File: index.js [hunk 2]
```diff
@@ -3,19 +3,25 @@ module.exports = TorrentGraph
...
 var debounce = require('debounce')
+var prettierBytes = require('prettier-bytes')
 
 var COLORS = {
-  links: '#FAFAFA',
+  links: {
+    color: '#C8C8C8',
+    width: 0.7, // default link thickness
+    maxWidth: 5.0, // max thickness
+    maxBytes: 2097152 // link max thickness at 2MB
```
Comment (feross) [inline index.js:13]: These aren't colors, so they don't really belong in this object.

File: index.js [hunk 3]
```diff
@@ -3,19 +3,25 @@ module.exports = TorrentGraph
...
+    subtitle: '#C8C8C8'
   },
   nodes: {
     method: function (d, i) {
       return d.me
-        ? d3.hsl(210, 0.7, 0.725) // blue
-        : d.seeder
-          ? d3.hsl(120, 0.7, 0.725) // green
-          : d3.hsl(55, 0.7, 0.725) // yellow
+          ? d3.hsl(210, 0.7, 0.725) // blue
```
Comment (feross) [inline index.js:21]: All this indentation is very strange. I'm surprised that it passes `standard`.

Comment (feross) [reply index.js:21]: This should all just be 2 space indentation.

File: README.md (hunk 1 above)
Comment (roccomuso) [reply README.md:78]: Ok, I'm fixing it. :+1:

File: index.js (hunk 2 above)
Comment (roccomuso) [reply index.js:13]: better have a new object called `STYLE` pheraps?

File: index.js (hunk 3 above)
Comment (roccomuso) [reply index.js:21]: ok no problem!

File: index.js (hunk 2 above)
Comment (feross) [reply index.js:13]: Sure that works.

**Pull request merged**